contents/handbook/company/goal-setting.md

```diff
@@ -6,31 +6,40 @@ showTitle: true
 
 ## How it works
 
-Every [small team](../people/team-structure/why-small-teams) has:
-
-- As few Objectives as possible - ideally 1
-- Motivation - explains why the objective is set
-- Key Results that show if we're en route to achieving an Objective
+1. Every three months, a couple of weeks before the end of the period, James and Tim (+ other execs) come up with the company goals.
```

Comment:
I know we explicitly changed this from James H being responsible, but I think sharing this is what caused the process to be messy in the first place. Are we comfortable stating one person who is actually responsible for making this happen?

Reply:
Yeah happy for James H to be the responsible person here